com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -1226,10 +1226,11 @@ internal void UpdateAnimationState(AnimationState animationState)
                         NetworkLog.LogError($"[DestinationState To Transition Info] Layer ({animationState.Layer}) sub-table does not contain destination state ({animationState.DestinationStateHash})!");
                     }
                 }
-                else if (NetworkManager.LogLevel == LogLevel.Developer)
-                {
-                    NetworkLog.LogError($"[DestinationState To Transition Info] Layer ({animationState.Layer}) does not exist!");
-                }
+                // For reference, it is valid to have no transition information
+                //else if (NetworkManager.LogLevel == LogLevel.Developer)
```

Comment:
I think it's maybe best to remove the commented out code and simply leave the comment?

Reply:
Yeah... I was thinking that too but wanted to keep it here until we do a full overhaul of `NetworkAnimator`.